app/src/main/java/ch/epfl/sweng/studyup/firebase/Firestore.java

```diff
@@ -217,9 +219,9 @@ public void onComplete(@NonNull Task<QuerySnapshot> task) {
                         Boolean questionTrueFalse = (Boolean) questionData.get(FB_QUESTION_TRUEFALSE);
                         int questionAnswer = Integer.parseInt((questionData.get(FB_QUESTION_ANSWER)).toString());
                         String questionCourseName = Course.SWENG.name(); //questionData.get(FB_COURSE).toString();
+                        String questionLang = (String) questionData.get(FB_QUESTION_LANG);
```

Comment:
As a security against nullpointer, Don't you want to check if the field is present on the db and put "en" if it is not. Like this even the old questions would not crash our program :)

Reply:
I agree

Reply:
Oh nice one!